controllers/ListController.php

```diff
@@ -25,7 +25,8 @@ protected function renderGallery($items)
             'galleries' => $items,
             'canWrite' => $this->contentContainer->can(WriteAccess::class),
             'isAdmin' => $this->isAdmin(),
-            'showMore' => !$this->isLastPage()
+            'showMore' => !$this->isLastPage(),
+            'sortByCreated' => $this->getSettings()->sortByCreated
```

Comment:
no need to pass `sortByCreated`, because it is not used in the template `gallery_list`.

Reply:
Thanks, this was left over from me testing and learning.